includes/functions/llms.functions.templates.dashboard.php

```diff
@@ -298,7 +312,19 @@ function lifterlms_template_my_memberships_loop( $student = null ) {
 
 		if ( ! $memberships ) {
 
-			printf( '<p>%s</p>', __( 'You are not enrolled in any memberships.', 'lifterlms' ) );
+			printf(
+				'<p>%s</p>',
+				/**
+				 * Not enrolled text.
+				 *
+				 * Allows developers to filter the text to be displayed when the student is not enrolled in any memberships.
+				 *
+				 * @since [version]
+				 *
+				 * @param string $not_enrolled_text The text to be displayed when the student is not enrolled in any memberships.
+				 */
+				apply_filters( 'lifterlms_not_enrolled_text', esc_html__( 'You are not enrolled in any memberships.', 'lifterlms' ) )
```

Comment:
This hook the way it is right now has the same name of the courses one, so I would give some more context here too, e.g.
'lifterlms_dashboard_memberships_not_enrolled_text'

Reply:
Oops 🤦
Fixed it. Thanks, Rocco.